examples/bri-3/src/bri/zkp/models/proof.ts

```diff
@@ -0,0 +1,36 @@
+import { BpiAccount } from '../../identity/bpiAccounts/models/bpiAccount';
+
+export class Proof {
```

Comment:
we are not using _, and getter functions anymore.

Reply:
+1